### ISSUE TITLE
Android TV

### ISSUE THREAD

**Issue by manu3008** [TriPSs/popcorn-native#48]
Would it be possible to create an Android TV version? Your version of popcorn time is by far superior/updated to any other option out there right now, and I'm using it quite a bit.

**Comment by TriPSs** [comment]
This is something i'm planning to add in the future.

**Comment by manu3008** [comment]
I'm not much of a programmer but if I can help, then just tell me. 👍

**Comment by manu3008** [comment]
The casting to tv is working but the sound isn't, is it normal?

**Comment by TriPSs** [comment]
Not that i know of, you are casting to a Android TV or chromecast?

**Comment by manu3008** [comment]
To an Android TV.

**Comment by TriPSs** [comment]
Do you have this issue with every movie / show? I don't have a Android TV so i can't check it.

**Comment by manu3008** [comment]
Yeah, there's a high pitched intermittent noise so it's basically unwatchable. My guess is it's coming from the app because other things I cast work great.

**Comment by TriPSs** [comment]
Small update on this matter, i have a local version working so there is progress :)

**Comment by manu3008** [comment]
Great!! If you need any testing I'd be happy to help.

**Comment by manu3008** [comment]
I closed it by accident, how's the new version going?

**Comment by TriPSs** [comment]
@manu3008 Did not continue working on it yet, still have to figure out a good design that works on tv

**Comment by manu3008** [comment]
Great, thanks and you just let me know if I can help in any way

**Comment by TriPSs** [comment]
@manu3008 had some inspiration, what do you think?
![image](https://user-images.githubusercontent.com/7668692/64550096-6baf8000-d332-11e9-87d2-c5b9ca8a5247.png)

Navigating in it does not work yet, have to figure out how to do that properly.

**Comment by manu3008** [comment]
Looks amazing man, I like it way better than how popcorn time manages it with the cross navigation. 👍👍
